src/components/ContentPreview/ContentPreview.js

```diff
@@ -1145,4 +1147,4 @@ class ContentPreview extends PureComponent<Props, State> {
 
 export type ContentPreviewProps = Props;
 export { ContentPreview as ContentPreviewComponent };
-export default withErrorBoundary(ORIGIN_CONTENT_PREVIEW)(makeResponsive(ContentPreview));
+export default withErrorBoundary(ORIGIN_CONTENT_PREVIEW)(makeResponsive(withFeatureProvider(ContentPreview)));
```

Comment:
We may want to consider adding something akin to redux's [compose](https://github.com/reduxjs/redux/blob/master/src/compose.js) utility.

Reply:
_.flow would work but it would probably still look weird with such a long line length https://lodash.com/docs/4.17.11#flow